.github/workflows/beam_LoadTests_Python_Combine_Flink_Streaming.yml

```diff
@@ -0,0 +1,128 @@
+      matrix:
+        job_name: ["beam_LoadTests_Python_Combine_Flink_Streaming"]
+        job_phrase: ["Run Load Tests Python Combine Flink Streaming"]
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup repository
+        uses: ./.github/actions/setup-action
+        with:
+          comment_phrase: ${{ matrix.job_phrase }}
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          github_job: ${{ matrix.job_name }} (${{ matrix.job_phrase }})
+      - name: Setup environment
+        uses: ./.github/actions/setup-environment-action
+        with:
+          python-version: 3.8
+      - name: Prepare test arguments
+        uses: ./.github/actions/test-arguments-action
+        with:
+          test-type: load
+          test-language: python
+          argument-file-paths: |
+            ${{ github.workspace }}/.github/workflows/load-tests-job-configs/python_Combine_Flink_Streaming_2GB_Fanout_4.txt
+            ${{ github.workspace }}/.github/workflows/load-tests-job-configs/python_Combine_Flink_Streaming_2GB_Fanout_8.txt   
+      - name: Start Flink with parallelism 16
+        env:
+          FLINK_NUM_WORKERS: 16
+        run: |
+          cd ${{ github.workspace }}/.test-infra/dataproc; ./flink_cluster.sh create
+      - name: get current time
+        run: echo "NOW_UTC=$(date '+%m%d%H%M%S' --utc)" >> $GITHUB_ENV
+      # The env variables are created and populated in the test-arguments-action as "<github.job>_test_arguments_<argument_file_paths_index>"  
+      - name: run Load test 2GB Fanout 4
+        uses: ./.github/actions/gradle-command-self-hosted-action
+        with:
+          gradle-command: :sdks:python:apache_beam:testing:load_tests:run
+          arguments: |
+            -PpythonVersion=3.8 \
+            -PloadTest.mainClass=apache_beam.testing.load_tests.combine_test \
+            -Prunner=PortableRunner \
+            '-PloadTest.args=${{ env.beam_LoadTests_Python_Combine_Flink_Streaming_test_arguments_1 }} --job_name=load-tests-python-flink-streaming-combine-4-${{env.NOW_UTC}}' \
+      - name: run Load test 2GB Fanout 8
+        uses: ./.github/actions/gradle-command-self-hosted-action
+        with:
+          gradle-command: :sdks:python:apache_beam:testing:load_tests:run
+          arguments: |
+            -PpythonVersion=3.8 \
+            -PloadTest.mainClass=apache_beam.testing.load_tests.combine_test \
+            -Prunner=PortableRunner \
+            '-PloadTest.args=${{ env.beam_LoadTests_Python_Combine_Flink_Streaming_test_arguments_2 }} --job_name=load-tests-python-flink-streaming-combine-5-${{env.NOW_UTC}}' \
+      - name: Teardown Flink
+        if: always()
+        run: |
+          ${{ github.workspace }}/.test-infra/dataproc/flink_cluster.sh delete
+
+        # // TODO(https://github.com/apache/beam/issues/20402): Skipping some cases because they are too slow.
+        # def TESTS_TO_SKIP = [
+        #   'load-tests-python-flink-streaming-combine-1',
+        # ]
```

Comment:
```suggestion
        # // TODO(https://github.com/apache/beam/issues/20402). Skipping some cases because they are too slow:
        # load-tests-python-flink-streaming-combine-1'
```

Reply:
updated, thanks